content/slides/data-formats-and-inspection.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Data/metadata inspection"
+title: "Geospatial data formats and metadata inspection"
```

Comment:
This title is a bit much for the slide deck :laughing: 

![image](https://github.com/qgreenland-workshop-2023-researcher/qgreenland-workshop-2023-researcher.github.io/assets/18539827/df434fe2-ac52-47e0-b0e7-f932c2fe194e)